.editorconfig

```diff
@@ -0,0 +1,12 @@
+root = true
+
+[*]
+end_of_line = lf
+indent_style = space
+insert_final_newline = true
+
+[*.{css,html,js,json,jsx,scss,yml,yaml}]
+indent_size = 2
+
+[*.{purs}]
+indent_size = 4
```

Comment:
Nit: Looks like your own editor doesn’t respect `insert_final_newline` 😜

Reply:
That's exactly what I added `.editorconfig` for